### PR TITLE
refactor: improve supergraph rule names

### PR DESCRIPTION
**Pull request by vindard**

### CODE REVIEW

File: dev/BUCK [hunk 1]
```diff
@@ -66,13 +66,13 @@ test_suite(
...
 dev_update_file(
-  name = "update-supergraph",
+  name = "update-supergraph-with-existing-schemas",
   generated = ":supergraph",
   out = "config/apollo-federation/supergraph.graphql"
 )
 
 sh_binary(
-  name = "update-schemas",
+  name = "update-schemas-and-supergraph",
```
Comment (nicolasburtey) [inline dev/BUCK:75]: I think there should be the notion of `update-graph-apps` versus `update-graph-core` to make it more intelligible what part those rules are addressing

Comment (nicolasburtey) [reply dev/BUCK:75]: I think if we have a `update-generated`, maybe it would more clear that there is other buck2 flow

Comment (bodymindarts) [reply dev/BUCK:75]: There should only be 1 command. BUCK figures or the order. 
i agree that we need a similar setup for Gen Code stuff.

**Comment by vindard** [comment]
This was addressed in #4260

**Pull request closed without merging**